glTF-Toolkit/src/GLTFLODUtils.cpp

```diff
@@ -360,9 +374,12 @@ namespace
                                            localMaterial.metallicRoughness.baseColorFactor == globalMaterial.metallicRoughness.baseColorFactor &&
                                            localMaterial.metallicRoughness.metallicFactor == globalMaterial.metallicRoughness.metallicFactor &&
                                            localMaterial.occlusionTexture.strength == globalMaterial.occlusionTexture.strength &&
-                                           localMaterial.specularGlossiness.diffuseFactor == globalMaterial.specularGlossiness.diffuseFactor &&
-                                           localMaterial.specularGlossiness.glossinessFactor == globalMaterial.specularGlossiness.glossinessFactor &&
-                                           localMaterial.specularGlossiness.specularFactor == globalMaterial.specularGlossiness.specularFactor;
+                                           localMaterial.HasExtension<KHR::Materials::PBRSpecularGlossiness>() == globalMaterial.HasExtension<KHR::Materials::PBRSpecularGlossiness>() && 

```

Comment:
What's the reason for not using PBRSpecularGlossiness.IsEqual(other)?

Reply:
Ok, in that case can we reduce the number of GetExtension calls to 2? One for localMaterial and one for globalMaterial.

Reply:
That would make this expression a lot more complex (looking).; I'll leave the optimization up to the compiler.